starknet_py/net/account/account_proxy.py

```diff
@@ -0,0 +1,102 @@
+from typing import List, Optional
+
+from starknet_py.net import AccountClient
+from starknet_py.net.account.base_account import BaseAccount
+from starknet_py.net.client import Client
+from starknet_py.net.client_models import (
+    Calls,
+    SentTransactionResponse,
+    Declare,
+    InvokeFunction,
+)
+from starknet_py.net.models import AddressRepresentation
+from starknet_py.net.models.transaction import DeployAccount
+from starknet_py.net.models.typed_data import TypedData
+
+
+class _AccountProxy(BaseAccount):
```

Comment:
_ should not be used in class name.

Reply:
But I want to indicate it's a private class and users should not ever use it (since it may change without any deprecations).

Reply:
I know, but it breaks the naming convention. I think it shouldn't be a problem if we don't document it and a comment here.

Reply:
But if keeping to convention is more important in your opinion, we can change it. I don't have that strong opinion on that matter.

Reply:
There are better ways of doing that. You should prefix the file name with `_` and set `__all__` to empty list (https://peps.python.org/pep-0008/#public-and-internal-interfaces).

Reply:
I guess we can do that, perhaps in other places as well. Should we create an issue for it?

Reply:
Yes, let's do it like with the keyword only arguments and start here.